src/3.x/package-development.md

```diff
@@ -94,9 +94,9 @@ Product::macro('myTestFunction', function () {
 });
 ```
 
-### Adding a multiple functions
+### Adding multiple functions
 
-If you want to add multiple functions this might get cluttered, in that case you can use a mixin; all functions defined in there will be made available from your model:
+If you want to add multiple functions this might get cluttered, in that case, you can use a mixin; all functions defined in there will be made available from your model:
```

Comment:
```suggestion
If you want to add multiple functions, this might get cluttered. In that case, you can use a mixin. All functions defined in there will be made available from your model:
```